django/publicmapping/publicmapping/settings.py

```diff
@@ -0,0 +1,199 @@
+"""
+Django settings for publicmapping project.
+
+Generated by 'django-admin startproject' using Django 1.11.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.11/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.11/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '1%a60e98ywm#$gl4mcm%s#uxoppc3)t%w+an*gzmujo-v-i07^'
```

Comment:
It might be good to change this or deliberately leave it out so that it doesn't slip through the cracks as we work on this further.

Reply:
Yeah, I think that we can pull it from the environment such that it leaves this setting unset if we forget to populate it:

```python
SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
```